go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/stackrox/rox
 
-go 1.19
+go 1.21
```

Comment:
do we do anything/depend on anything which requires 1.21?

Reply:
`go mod tidy` adds it automatically :(